domain/access/bootstrap/bootstrap_test.go

```diff
@@ -24,7 +24,7 @@ var _ = gc.Suite(&bootstrapSuite{})
 
 func (s *bootstrapSuite) SetUpTest(c *gc.C) {
 	s.ControllerSuite.SetUpTest(c)
-	s.controllerUUID = s.SeedControllerUUID(c)
+	s.controllerUUID = s.SeedControllerTable(c, "")
```

Comment:
You can use this https://github.com/juju/juju/blob/f90d2379b4ee14bcadf62ee29c70fec1a65d27bf/internal/testing/environ.go#L52 to seed the database

Reply:
My thinking was that it would be good to know which model is the controller model. In a lot of tests a model is created and this is used as the UUID, if we separately populate this in the controller model table then we might end up with things working when perhaps they shouldn't due to these coincidentally being the same.

If you think there is a good reason to use that value that can I can change it.

Reply:
We can have another constant in that file. Something akin to `ControllerModelTag`. I just want a value, don't care what.

Reply:
Ok, will add one